index.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+
```

Comment:
- [ ] As suggested by the previous reviewer, kindly add the white background as it will make your design stick to the Figma .

![Screenshot from 2023-03-04 15-53-46](https://user-images.githubusercontent.com/97834160/222906494-4695301f-5e36-46d8-b3e8-76caa0d18e5a.png)

Reply:
Background color has been changed